tests/spec/w3c/headers-spec.js

```diff
@@ -231,6 +250,19 @@ describe("W3C — Headers", function() {
       const doc = await makeRSDoc(ops);
       expect($("#subtitle", doc).length).toEqual(0);
     });
+    
+    it("uses existing h2#subtitle as subtitle", async () => {
+      const ops = makeStandardOps();
+      ops.body = "<h2 id='subtitle'>This is a pre existing subtitle</h2>" + makeDefaultBody();
```

Comment:
Make sure you add another element (e.g., `<code>pass</code>`) inside the `h2`, as that's the purpose of the change.

Reply:
Sure. The other changes belong to @sidvishnoi I guess. So I hope he would correct them :+1:

Reply:
@shikhar-scs on it.